app/libs/dto/api_params_adapter.rb

```diff
@@ -33,7 +33,7 @@ def map_week_slugs_to_weeks
         end
         params[:weeks] = concatenated_query.all
       else
-        params[:weeks] = Week.selectable_from_now_until_end_of_period
+        params[:weeks] = Week.selectable_from_now_until_school_year
```

Comment:
wondering about this one... on create should be upcoming ; on edit... maybe we should simply not touch it ?

Reply:
modifier uniquement si c'est specifié